infoschema/infoschema.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/model"
 	"github.com/pingcap/tidb/table"
+	_ "github.com/pingcap/tidb/table/tables"
```

Comment:
golint reports 
"infoschema.go:25:2: a blank import should be only in a main or test package, or have a comment justifying it"